README.rst

```diff
@@ -75,12 +75,15 @@ support systems:
   the Github issues system: https://github.com/zephyrproject-rtos/zephyr/issues.
   You can browse through the reported issues and submit issues of your own.
 
-* **Security-related Issue Reporting**: For security-related inquiries or
-  reporting suspected security-related bugs in the Zephyr OS, please
-  send email to vulnerabilities@zephyrproject.org.  We will assess and fix
-  flaws according to our security policy outlined in the Zephyr Project
+* **Security-related Issue Reporting and Tracking**: For security-related
+  inquiries or reporting suspected security-related bugs in the Zephyr OS,
+  please send email to vulnerabilities@zephyrproject.org.  We will assess and
+  fix flaws according to our security policy outlined in the Zephyr Project
   `Security Overview`_.
 
+  Security related issue tracking is done in JIRA.  The location of this JIRA
+  is https://zephyrprojectsec.atlassian.net.
```

Comment:
Looks like you'll need to create an account to get in to this Jira instance (even for just viewing). You should mention that here.

Reply:
I was thinking of adding all of that on the jira website, but i could add it here as well

Reply:
Can you allow browsing Jira without logging in? That would be nice.

Reply:
I tried creating an account and got an error validating my email. Maybe it's not set up yet?

Reply:
ah let me see if i can change that setting

Reply:
ok dashboard is visible by anyone and I also allowed anyone to create a ticket.  Also configured this to allow self sign up.  So that should work now.